core/app/models/spree/stock_transfer.rb

```diff
@@ -100,6 +104,18 @@ def close(closed_by)
 
     private
 
+    def ensure_transfer_items_valid
```

Comment:
I don't like that as it has a potential side effect (adding errors to the model) which IMO isn't expected with ? methods